### ISSUE TITLE
BatchNorm is slow

### ISSUE THREAD

**Issue by fchollet** [keras-team/keras-core#225]
Especially when `jit_compile=False`, Keras Core BN is significantly slower than tf.keras BN. Need to fix.
